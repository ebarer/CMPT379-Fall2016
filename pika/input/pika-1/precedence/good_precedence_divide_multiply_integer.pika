exec {

  print 76 / 44 * 45, _n_.
  print 78 / 40 * 75, _n_.
  print 60 / 45 * 99, _n_.
  print 21 / 11 * 78, _n_.
  print 23 / 17 * 90, _n_.
  print 8 / 24 * 90, _n_.
  print 77 / 19 * 33, _n_.
  print 68 / 72 * 11, _n_.
  print 27 / 74 * 71, _n_.
  print 19 / 91 * 54, _n_.
  print 21 / 69 * 67, _n_.
  print 30 / 66 * 64, _n_.

  print 36 * 41 / 83, _n_.
  print 40 * 29 / 47, _n_.
  print 24 * 4 / 39, _n_.
  print 27 * 32 / 3, _n_.
  print 55 * 90 / 86, _n_.
  print 87 * 75 / 44, _n_.
  print 48 * 35 / 10, _n_.
  print 83 * 1 / 26, _n_.
  print 27 * 39 / 4, _n_.
  print 79 * 61 / 22, _n_.
  print 90 * 29 / 24, _n_.
  print 76 * 46 / 13, _n_.

  print 52 / 66 * 71 / 71, _n_.
  print 48 / 66 * 82 / 25, _n_.
  print 38 / 46 * 1 / 80, _n_.
  print 79 / 84 * 7 / 49, _n_.
  print 66 / 86 * 73 / 20, _n_.
  print 40 / 18 * 79 / 40, _n_.
  print 20 / 55 * 94 / 82, _n_.
  print 80 / 67 * 69 / 88, _n_.
  print 55 / 41 * 30 / 3, _n_.
  print 78 / 13 * 15 / 27, _n_.
  print 10 / 57 * 62 / 22, _n_.
  print 87 / 38 * 57 / 46, _n_.

  print 44 * 90 / 96 * 93, _n_.
  print 49 * 43 / 57 * 62, _n_.
  print 19 * 89 / 94 * 46, _n_.
  print 26 * 99 / 69 * 50, _n_.
  print 55 * 99 / 95 * 30, _n_.
  print 55 * 40 / 12 * 53, _n_.
  print 76 * 56 / 87 * 73, _n_.
  print 89 * 31 / 71 * 17, _n_.
  print 92 * 94 / 87 * 35, _n_.
  print 56 * 4 / 52 * 42, _n_.
  print 68 * 74 / 47 * 86, _n_.
  print 33 * 4 / 73 * 2, _n_.

}

