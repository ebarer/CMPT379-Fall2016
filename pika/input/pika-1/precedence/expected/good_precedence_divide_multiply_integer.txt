45
75
99
78
90
0
132
0
0
0
0
0
17
24
2
288
57
148
168
3
263
219
108
268
0
0
0
0
0
3
0
0
10
3
0
2
3813
2232
782
1850
1710
9699
3504
646
3465
168
9202
2
